src/ceph_fuse.cc

```diff
@@ -97,6 +97,14 @@ int main(int argc, const char **argv, const char *envp[]) {
       filer_flags |= CEPH_OSD_FLAG_LOCALIZE_READS;
     } else if (ceph_argparse_flag(args, i, "-h", "--help", (char*)nullptr)) {
       usage();
+    } else if (ceph_argparse_flag(args, i, "-V", "--version", (char*)nullptr)) {
+      struct fuse_args fargs = FUSE_ARGS_INIT(argc, (char**)argv);
+      if (fuse_parse_cmdline(&fargs, nullptr, nullptr, nullptr) == -1) {
+       derr << "fuse_parse_cmdline failed." << dendl;
+      }
+      assert(fargs.allocated);
+      fuse_opt_free_args(&fargs);
+      exit(1);
```

Comment:
`exit(0)`.

Reply:
Fixed. 

(I was wondering why `generic_client_usage()` and `generic_server_usage()` do `exit(1)`.)

Reply:
I don't know but it doesn't really make sense.

Reply:
Agreed.